CHANGELOG.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+.. _v37-0-1:
+
+37.0.1 - 2022-04-27
+~~~~~~~~~~~~~~~~~~~
+
+* Fixed an issue where parsing an encrypted private key with the public
+  loader functions would hang waiting for user input on OpenSSL 3.0.x rather
```

Comment:
I would use the word "console input" here for clarity.

Reply:
```suggestion
  loader functions would hang waiting for console input on OpenSSL 3.0.x rather
```